hostsflow/README

```diff
@@ -0,0 +1,18 @@
+Demo for generating flow from a Linux host and displaying it in New Relic.
+
+1) Create a free account at https://newrelic.com/.
+
+2) Follow the guided install for "Network Flow". Copy the account ID and API Key generated.
+
+3) Run with
+
+sudo NETWORK=$MY_NETWORK NR_INSIGHTS_INSERT_KEY=$MY_KEY NR_ACCOUNT_ID=$MY_ACCOUNT_ID docker-compose up
```

Comment:
replace: `NR_INSIGHTS_INSERT_KEY=$MY_KEY`

with `NR_LICENSE_KEY=$MY_KEY`

Reply:
ok -- should be good now.